ui/src/app/workflow-templates/workflow-template-details.tsx

```diff
@@ -65,6 +70,16 @@ export function WorkflowTemplateDetails({history, location, match}: RouteCompone
             .catch(setError);
     }, [name, namespace]);
 
+    useEffect(() => {
+        (async () => {
+            const workflowList = await services.workflows.list(namespace, null, [`${models.labels.workflowTemplate}=${name}`], {limit: 50});
+            const workflowsInfo = await services.info.getInfo();
+
+            setWorkflows(workflowList.items);
+            setColumns(workflowsInfo.columns);
+        })();
+    }, []);
```

Comment:
we may want to consider adding `name` as a hook dep in the array here, but this is identical to `cron-workflow-details.tsx` right now, so this is fine.